src/System.CommandLine/Command.cs

```diff
@@ -10,12 +10,29 @@
 
 namespace System.CommandLine
 {
-    public class Command : Symbol, ICommand, IEnumerable<Symbol>
+    public class Command : 
+        Symbol, 
+        ICommand, 
+        IEnumerable<Symbol>
     {
         private readonly SymbolSet _globalOptions = new SymbolSet();
 
-        public Command(string name, string? description = null) : base(new[] { name }, description)
+        public Command(string name, string? description = null) : base(description)
         {
+            Name = name;
+
+            AddAlias(Name);
+        }
+
+        public override string Name
+        {
+            // FIX: (Name) 
+            get => base.Name;
+            set
+            {
+                base.Name = value;
+                AddAlias(Name);
```

Comment:
Does the original base.Name need to be removed from the aliases? Like you do in the Options?

Reply:
Never mind. It looks like it will get removed in the base (Symbol) class's Name property.